### PR TITLE
Zib2020-MedicalDevice-HearingFunction-VisualFunction

### PR DESCRIPTION
**Pull request by LuudSlagter**

### CODE REVIEW

File: ada_2_fhir-r4/zibs2020/payload/0.0/nl-core-MedicalDevice.xsl [hunk 1]
```diff
@@ -0,0 +1,369 @@
...
+                <xsl:for-each select="indicatie">
+                    <xsl:call-template name="makeReference">
+                        <xsl:with-param name="in" select="probleem"/>
+                        <xsl:with-param name="wrapIn" select="'reasonReference'"/>
+                    </xsl:call-template>
+                </xsl:for-each>
+                
+                <!--The element reasonReference is present to support Observations that refer to a MedicalDevice, such as HearingFunction and VisualFunction. -->
+                <xsl:if test="$reasonReference">
+                    <xsl:for-each select="../..">
```
Comment (LuudSlagter) [inline ada_2_fhir-r4/zibs2020/payload/0.0/nl-core-MedicalDevice.xsl:146]: I wonder if this works in general (for HearingFunction and VisualFunction it works) or if some ada instances that refer to a MedicalDevice have another structure. Should we implement this as a variable instead, or check each time a mapping is made that includes a MedicalDevice whether this logic is still appropriate?

Comment (jduwel) [reply ada_2_fhir-r4/zibs2020/payload/0.0/nl-core-MedicalDevice.xsl]: Fixed

**Review by jduwel — APPROVED** [review]
Kind of approving my own changes. @LuudSlagter will have a final look

**Comment by LuudSlagter** [comment]
Everything looks fine, ready to merge!

**Pull request merged**